ecs-cli/Gopkg.toml

```diff
@@ -69,7 +69,7 @@
 
 [[constraint]]
   name = "github.com/stretchr/testify"
-  revision = "2402e8e7a02fc811447d11f881aa9746cdc57983"
+  version = "1.2.1"
```

Comment:
can I ask what motivated the upgrade? (new functionality available/needed?)

Reply:
`assert.ElementsMatch` (See: https://godoc.org/github.com/stretchr/testify/assert#ElementsMatch). Used [here](https://github.com/aws/amazon-ecs-cli/pull/507/files#diff-3049fccece183947d0bb4efc94eb922bR274) (and could potentially be used elsewhere -- before I upgraded I actually simplified some of the assertions on lists that had more than one element).

Reply:
nice!